kolibri/core/assets/src/views/StorageNotification.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+
+  <div v-if="bannerOpened" class="banner" :style="{ background: $themeTokens.surface }">
+    <div class="banner-inner">
+      <h1 style="display: none">
+        {{ $tr('bannerHeading') }}
+      </h1>
+      <KGrid>
+        <!-- Grid Content -->
+        <KGridItem :layout12="{ span: 12 }">
+          <p>
+            {{ getMessage() }}
+          </p>
+        </KGridItem>
+
+        <!-- Grid Buttons -->
+        <KGridItem class="button-grid-item" :layout12="{ span: 12 }">
+          <KButton
+            :text="coreString('closeAction')"
+            style="margin-right: 10px"
+            appearance="flat-button"
+            :secondary="true"
+            @click="closeBanner"
+          />
+          <KButton
+            v-if="availableDownloads"
+            :text="$tr('goToDownloads')"
+            appearance="raised-button"
+            :secondary="true"
+            @click="closeBanner"
+          />
+          <KButton
+            v-if="isAdmin"
+            :text="$tr('manageChannels')"
+            appearance="raised-button"
+            :secondary="true"
+            @click="manageChannel"
+          />
+        </KGridItem>
+      </KGrid>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import { mapGetters } from 'vuex';
+  import plugin_data from 'plugin_data';
+
+  export default {
+    name: 'StorageNotification',
+    components: {},
+    mixins: [commonCoreStrings],
+    props: {
+      showBanner: {
+        type: String,
+        required: false,
+        default: null,
+      },
+    },
+    data() {
+      return {
+        isSubsetOfUsersDevice: plugin_data.isSubsetOfUsersDevice,
```

Comment:
Is there an existing example you took this from? I haven't see something like this

Reply:
Oh nevermind, in the PR deletions there's an example :+1: